dvc/utils/conversions.py

```diff
@@ -0,0 +1,24 @@
+# https://github.com/aws/aws-cli/blob/5aa599949f60b6af554fd5714d7161aa272716f7/awscli/customizations/s3/utils.py
```

Comment:
Since this whole file is awscli-specific, we could move this stuff to `dvc/tree/s3.py` or create something like `dvc/tree/s3/utils.py` and it put it there. This piece is quite small, so could go with the former no problem.

Reply:
I think keeping here is OK (even though this is a pre-emptive assumption) I believe this utility might come in handy in other places since it does a very general job. The awscli-specific part here is the handling of IEC suffixes, though it doesn't differ much and something we can ignore.

Reply:
Ok, let's keep it then :+1: